src/views/partials/breaches.js

```diff
@@ -76,6 +76,26 @@ function createResolveSteps (breach) {
   return resolveStepsHTML.join('')
 }
 
+/**
+ * @param {*} data
+ * @param {'none' | 'all-resolved'} status
+ * @returns string
+ */
+function createAddEmailButton (data, status) {
+  if (data.emailCount >= AppConstants.MAX_NUM_ADDRESSES) {
+    return ''
+  }
+
+  return `
+    <p>
+      ${getMessage(`breaches-${status}-cta-blurb`)}
+    </p>
+    <button class="primary" data-dialog='add-email'>
```

Comment:
I just reused the `primary` class, but in the designs, it uses the Metropolis font and a font weight of 600. Should I update the `primary` class, or change it just for this instance?

Reply:
We probably want to stay consistent with the buttons. I’d advocate for reusing the `primary` class for now and we can leave a comment for UX to see if they can adjust the design accordingly or otherwise change it respectively.

Reply:
Yes, as my other comments have trended – instead of creating too many new unique styles, default to using our established vars for consistency and get UX sign-off during review.